app/uk/gov/hmrc/merchandiseinbaggagefrontend/controllers/ReviewGoodsController.scala

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.merchandiseinbaggagefrontend.controllers
+
+import javax.inject.{Inject, Singleton}
+import play.api.data.Form
+import play.api.mvc.{Action, AnyContent, MessagesControllerComponents}
+import uk.gov.hmrc.merchandiseinbaggagefrontend.config.AppConfig
+import uk.gov.hmrc.merchandiseinbaggagefrontend.forms.ReviewGoodsFormProvider
+import uk.gov.hmrc.merchandiseinbaggagefrontend.repositories.DeclarationJourneyRepository
+import uk.gov.hmrc.merchandiseinbaggagefrontend.views.html.ReviewGoodsView
+
+@Singleton
+class ReviewGoodsController @Inject()(override val controllerComponents: MessagesControllerComponents,
+                                      actionProvider: DeclarationJourneyActionProvider,
+                                      formProvider: ReviewGoodsFormProvider,
+                                      repo: DeclarationJourneyRepository,
+                                      view: ReviewGoodsView)
+                                     (implicit appConfig: AppConfig)
+  extends DeclarationJourneyUpdateController {
+
+  val form: Form[Boolean] = formProvider()
+
+  val onPageLoad: Action[AnyContent] = actionProvider.journeyAction { implicit request =>
```

Comment:
This controller and view should use the Goods case class rather than GoodEntries because the goods entries should be complete when the user browses here.

If there are any incomplete goods entries we should be directed back to complete them.

While we support a single goods item this means we just need to worry about the first goods item being complete or otherwise, and re-directing back as appropriate.

Reply:
I disagree, this is a Yes/No question so the form is fine being a Boolean. When the form is submitted we only need to know this value to decide where to redirect the user.

If they have any incomplete goods entries then we catch that in the controller before displaying the page

Reply:
We had an outbreak of consensus!